services/src/main/java/io/scalecube/services/ServiceErrors.java

```diff
@@ -0,0 +1,14 @@
+package io.scalecube.services;
+
+import io.scalecube.services.api.ErrorData;
+import io.scalecube.services.api.Qualifier;
+import io.scalecube.services.api.ServiceMessage;
+
+public interface ServiceErrors {
+
+  ServiceMessage EMPTY_RESPONSE_ERROR =
```

Comment:
Let's not have separate class for this, just def it as part of logic of a function as proposed in comment at [line 41](https://github.com/scalecube/scalecube-services/pull/450/files#diff-c402059b5663b50c40754f48b7677f9cR41)

Reply:
That could be a good place to collect different types of errors... ok, let's create it as we have the second error to collect there